exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/spill/RecordBatchSizer.java

```diff
@@ -65,6 +70,14 @@
 
     public int stdSize;
 
+    /**
+     * If the we can determine the exact width of the row of a vector upfront,
+     * the row widths is saved here. If we cannot determine the exact width
+     * (for example for VarChar or Repeated vectors), then
+     */
+
+    private int knownSize = -1;
```

Comment:
Like I mentioned in other comment, seems like we can just use stdSize.

Reply:
Thanks for catching this. I'll use stdSize instead.